front-end/src/components/main-page/MainPage.jsx

```diff
@@ -110,7 +122,7 @@ class MainPage extends Component {
                     aria-label="Enter Project ID"
                     placeholder="Enter Project ID"
                     onChange={this.handleChange}
-                    value={this.props.projectIdToBeDisplayed}
+                    value={this.props.projectIdToBeDisplayed.split("merge")[0]}
```

Comment:
Is there a way to simplify this a little? Could we use an object instead of a string?

Reply:
I thought of doing that but then I have to do a whole refactoring because we use projectIdToBeDisplayed everywhere we use store